.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Nuxt Community Discord
+    url: https://discord.nuxtjs.org/
+    about: Consider asking questions about the module here.
+  - name: {{ name }} documentation
```

Comment:
Can we maybe comment out documentation lines? Templates are not supported and modules usually only have a README as docs unless one creates, which as a chore task, can also enable/add this link

Reply:
Ok, I agree.

Reply:
Addressed. I commented out the docs section, but left in a "/README.md" as a default (commented out).